src/msw/webview_ie.cpp

```diff
@@ -853,25 +856,85 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
-void wxWebViewIE::RunScript(const wxString& javascript)
+bool wxWebViewIE::MSWSetModernEmulationLevel(bool modernLevel)
 {
-    wxCOMPtr<IHTMLDocument2> document(GetDocument());
-
-    if(document)
+    wxRegKey key(wxRegKey::HKCU, wxREGISTRY_IE_PATH);
+    if ( key.Exists() )
     {
-        wxCOMPtr<IHTMLWindow2> window;
-        wxString language = "javascript";
-        HRESULT hr = document->get_parentWindow(&window);
-        if(SUCCEEDED(hr))
+        wxString programName = wxGetFullModuleName().AfterLast('\\');
+        if ( modernLevel )
         {
-            VARIANT level;
-            VariantInit(&level);
-            V_VT(&level) = VT_EMPTY;
-            window->execScript(wxBasicString(javascript),
-                               wxBasicString(language),
-                               &level);
+            if ( !key.SetValue(programName, wxIE_EMULATION_LEVEL) )
+            {
+	            wxLogWarning(_("Failed to set the current browser control emulation level"));
```

Comment:
tabs.
Detected with the [Fix Mixed Tabs](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.FixMixedTabs) extension.